src/sentry/middleware/access_log.py

```diff
@@ -65,16 +67,17 @@ def _create_api_access_log(
     Create a log entry to be used for api metrics gathering
     """
     try:
+        resolver_match = request.resolver_match
```

Comment:
If for some reason `resolver_match` doesn't exist, the request will fail cause of this.

use `getattr` with a default to be safe.